### DISCUSSION TITLE
what this problem

### DISCUSSION THREAD

**Discussion by HalFrenj** [community/community#57829] [Programming Help]
### Body

import Image from 'next/image'
import { useState } from 'react'
import styles from './page.module.css'

export default function Home() {
  const [count, setCount] = useState(0);

  const onIncrease = () => {
    setCount(prevCount => prevCount + 1);
  };
  
  const onDecrease = () => {
    setCount(prevCount => prevCount - 1);
  };

  return (
    <div>
      <p>Count : {count}</p>
      <button onClick={onIncrease}>+</button>
      <button onClick={onDecrease}>-</button>
    </div>
  )
}

import Image from 'next/image'
 2 | import { useState } from 'react'
   :          ^^^^^^^^
 3 | import styles from './page.module.css'
 4 | 
 5 | export default function Home() {
   `----

Maybe one of these should be marked as a client entry with "use client":
  ./app/page.tsx

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by leonden** [comment]
It appears that you are trying to import the `Image` component from `next/image` and the `useState` hook from `react`. However, there seems to be an issue with your code. The `import` statement for `Image` is duplicated, which is causing the error. You should remove the duplicate import statement for `Image` and keep only one.

Here's the corrected code:

```tsx
import { useState } from 'react'
import styles from './page.module.css'

export default function Home() {
  const [count, setCount] = useState(0);

  const onIncrease = () => {
    setCount(prevCount => prevCount + 1);
  };

  const onDecrease = () => {
    setCount(prevCount => prevCount - 1);
  };

  return (
    <div>
      Count: {count}
      <button onClick={onIncrease}>+</button>
      <button onClick={onDecrease}>-</button>
    </div>
  );
}
```

In this corrected code, I removed the duplicated import statement for `Image`, and I wrapped the TSX elements inside a `<div>` for proper rendering. The count value is displayed, and two buttons are provided to increase and decrease the count.
